test/SlowTests/Server/Documents/ETL/Raven/RavenDB_3760.cs

```diff
@@ -54,7 +56,12 @@ public void Null_returned_from_script_means_that_document_is_filtered_out()
             {
                 var etlDone = WaitForEtl(master, (n, statistics) => statistics.LoadSuccesses != 0);
 
-                SetupEtl(master, slave, "users", @"if (this.Age % 2 == 0) return null; else this.Name = 'transformed'; return this;");
+                SetupEtl(master, slave, "users", @"
+if (this.Age % 2 == 0) 
+    return null; 
```

Comment:
It has no effect. Those tests were just migrated from 3.5 where returning `null` meant to filter a document out. We don't have such usage now.